src/pages/Pools.tsx

```diff
@@ -43,6 +50,11 @@ const Pools = ({ itemsPerPage }: IPoolsProps) => {
   const [userPoolsToShow, setUserPoolsToShow] = useState<IPoolExtendedData[]>([]);
   const [havePools, setHavePools] = useState(false);
   const [haveUserPools, setHaveUserPools] = useState(false);
+  const [poolsAnalytics, setPoolsAnalytics] = useState({
+    tvl: 0,
+    volume24h: 0,
```

Comment:
We could extract this object into initialPoolsAnalytics and reuse it into the reduce statement below

Reply:
@DimitarTAtanasov Done